lib/smack/SmackRep.cpp

```diff
@@ -55,6 +59,36 @@ bool SmackRep::isInt(const llvm::Value* v) {
   return isInt(v->getType());
 }
 
+unsigned SmackRep::getIntSize(const llvm::Value* v) {
+  assert(isInt(v) && "wrong type, should be integer.");
```

Comment:
In fact, this assertion should be omitted, since it is redundant with the subsequent `llvm::cast`.